Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -2139,6 +2189,15 @@ public class TimeSlider: UIControl {
             }
         }
         
+        if let changedIndexes = changedIndexes, let operationalLayers = geoView?.operationalLayers, !operationalLayers.isEmpty  {
```

Comment:
You could simplify this using `Collection.contains(where:)`:

```swift
if let changedIndexes = changedIndexes,
    let operationalLayers = geoView?.operationalLayers,
    changedIndexes.contains(where: { operationalLayers[$0] is AGSTimeAware }) {
    return true
}
```

Reply:
fixed